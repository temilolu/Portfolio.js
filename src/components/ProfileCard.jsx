import React, {Component} from 'react';
import { Link } from 'react-router-dom';
import Landing from './Landing';

const ProfileCard = (props) => (
     <div className="col-md-4">
        <div className="profile">   
            <div className="profile--img"> 
                <img src={props.image} alt={props.name} />
            </div>
            
            <div className="profile--bio">
                <h2>{props.name}</h2>
                <h4>{props.job}</h4>
                <ul className="list-inline">
                    <li><a href="https://twitter.com/temity09" target="_blank"><i className="fa fa-twitter" aria-hidden="true"></i></a></li>
                    <li><a href="https://github.com/temilolu" target="_blank"><i className="fa fa-github" aria-hidden="true"></i></a></li>
                    <li><a href="https://medium.com/@temity09" target="_blank"><i className="fa fa-medium" aria-hidden="true"></i></a></li>
                    <li><a href="https://www.linkedin.com/in/temitayo-ogunlolu/" target="_blank"><i className="fa fa-linkedin-square" aria-hidden="true"></i></a></li>
                </ul>
            </div>
        </div>
    </div>
 );

 ProfileCard.propTypes = {
     profile: React.PropTypes.shape({
         name: React.PropTypes.string.isRequired,
         job: React.PropTypes.string,
     })

 }
export default ProfileCard;