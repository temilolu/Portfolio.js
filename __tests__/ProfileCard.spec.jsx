import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import ProfileCard from '../src/components/ProfileCard';

test('Should Display User name and job ', () => {
    const output = shallow(<ProfileCard name={profile.name} />);
    expect(output).toMatchSnapshot();
})
