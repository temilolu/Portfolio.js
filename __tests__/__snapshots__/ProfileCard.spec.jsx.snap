// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Display User name and job  1`] = `
<div
  className="col-md-4"
>
  <div
    className="profile"
  >
    <div
      className="profile--img"
    >
      <img
        alt="ProfileCard"
      />
    </div>
    <div
      className="profile--bio"
    >
      <h2>
        ProfileCard
      </h2>
      <h4 />
      <ul
        className="list-inline"
      >
        <li>
          <a
            href=""
          >
            <i
              aria-hidden="true"
              className="fa fa-twitter"
            />
          </a>
        </li>
        <li>
          <a
            href=""
          >
            <i
              aria-hidden="true"
              className="fa fa-github"
            />
          </a>
        </li>
        <li>
          <a
            href=""
          >
            <i
              aria-hidden="true"
              className="fa fa-medium"
            />
          </a>
        </li>
        <li>
          <a
            href=""
          >
            <i
              aria-hidden="true"
              className="fa fa-linkedin-square"
            />
          </a>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
